### PR TITLE
Add github actions to build future releases

### PR DESCRIPTION
**Pull request by nepalez**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          --merge repository/index.yaml
+          repository/
+      - name: Commit & Push gh-pages with a new chart
+        run: |
+          cd repository/
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add --all
+          git commit -m "Release ${GITHUB_REF##*/} to gh-pages"
+          git push origin gh-pages
```
Comment (Envek) [inline .github/workflows/release.yml:36]: I believe that we can use one of existing actions for that (there are plenty of them: [1](https://github.com/marketplace/actions/github-pages), [2](https://github.com/marketplace/actions/deploy-to-github-pages), [3](https://github.com/marketplace/actions/github-pages-action)). For example:
```suggestion
      - name: Deploy pages
        uses: crazy-max/ghaction-github-pages@v2
        with:
          build_dir: repository
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

File: Makefile [hunk 2]
```diff
@@ -12,9 +12,3 @@ refresh: prepare
 build: prepare refresh
 	helm package -d repository/ anycable-go
 	helm repo index --url https://helm.anycable.io/ --merge repository/index.yaml repository/
-
-release: build
-	cd repository/ && \
-	git add --all && \
-	git commit -m "Release all charts to gh-pages via 'make release'" && \
-	git push origin gh-pages
```
Comment (Envek) [inline Makefile:20]: I believe that we can keep it as a backup (in case if GH Actions would break)

**Review by Envek — COMMENTED** [review]
IMO it is better to keep manual release process as a backup and document automatic releasing separately. E.g. “Release will be published automatically after pushing a tag, but here are manual steps just in case: …”

**Comment by nepalez** [comment]
@Envek thanks. fixed

**Review by deponian — APPROVED** [review]

**Comment by nepalez** [comment]
The PR is closed as stale

**Pull request closed without merging**